iTunesSearch/iTunesSearch/SearchResultController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchResultController.swift
+//  iTunesSearch
+//
+//  Created by Dojo on 7/8/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultController {
+    
+    let baseURL = URL(string: "https://itunes.apple.com/search")!
```

Comment:
I know we haven't talked much about access control, but Johnny is using it marking things as private where appropriate, so try to see if you can pick up on that a bit. For example this baseURL doesn't need to be accessed anywhere else in your app, so marking it private would be appropriate.